### PR TITLE
Add support for bonds to SDF format

### PR DESCRIPTION
**Pull request by tovrstra**
This PR adds support for the bond information in the SDF format (both load and dump). I've also made some minor improvements to conform better to the official format.

### CODE REVIEW

File: iodata/formats/sdf.py [hunk 1]
```diff
@@ -41,23 +47,36 @@
 PATTERNS = ['*.sdf']
 
 
-@document_load_one("SDF", ['atcoords', 'atnums', 'title'])
+SDF2BOND = {1: "1", 2: "2", 3: "3", 4: "ar", 5: "sd", 6: "sar", 7: "dar", 8: "un"}
+BOND2SDF = dict((val, key) for key, val in SDF2BOND.items())
```
Comment (evohringer) [inline iodata/formats/sdf.py:51]: Could you maybe leave a comment why you had to define this dicitionary locally and not use the one in periodic. Just to have it commented. :-)

**Review by evohringer — APPROVED** [review]
It looks all great @tovrstra. Only a small suggestion to add a comment.

File: iodata/formats/sdf.py (hunk 1 above)
Comment (tovrstra) [reply iodata/formats/sdf.py:51]: We may be able to get rid of these two mappings, which may also be a good solution. The reason for having them is that MOL2 uses slightly different conventions, but we can bend those a little.

Comment (tovrstra) [reply iodata/formats/sdf.py]: I've eliminated `SDF2BONDS` because it seemed better get rid of complicated things than having to explain them.

**Review by evohringer — APPROVED** [review]
This looks good to me!

**Comment by tovrstra** [comment]
@evohringer Thanks for reviewing!

**Pull request merged**